flink-docs/src/main/java/org/apache/flink/docs/configuration/ConfigOptionsDocGenerator.java

```diff
@@ -69,6 +71,8 @@
 	};
 
 	static final Set<String> EXCLUSIONS = new HashSet<>(Arrays.asList(
+		"org.apache.flink.configuration.ReadableConfig",
```

Comment:
is this actually related to the PR?

Reply:
Yes and no. This should've been part of the https://issues.apache.org/jira/browse/FLINK-14493
Unfortunately I was not aware of this logic and that the `ReadableConfig` and `WritableConfig` will be picked up by the documentation generator. Those are new read-only, write-only interfaces for `Configuration`. Config tables should not be generated for them.

Reply:
Then I'd move this change into a separate commit.